jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Fields.java

```diff
@@ -329,7 +330,31 @@ public Iterator<Field> iterator()
      */
     public Map<String, String[]> toStringArrayMap()
     {
-        Map<String, String[]> result = new LinkedHashMap<>();
+        Map<String, String[]> result = new LinkedHashMap<>()
+        {
+            @Override
+            public String toString()
+            {
+                if (fields.isEmpty())
+                    return "{}";
+
+                StringBuilder sb = new StringBuilder();
+                sb.append('{');
+                for (Iterator<Map.Entry<String, String[]>> i = entrySet().iterator(); i.hasNext();)
+                {
+                    Map.Entry<String, String[]> e = i.next();
+                    String key = e.getKey();
+                    String[] value = e.getValue();
+                    sb.append(key);
+                    sb.append('=');
+                    sb.append(Arrays.asList(value));
+                    if (i.hasNext())
+                        sb.append(',');
+                }
+                sb.append('}');
+                return sb.toString();
```

Comment:
Can we have a single implementation, a static method somewhere in jetty-util that does this?
That way this location, and MultiMap can use the same one?